src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form
+      action="Post"
+      class="form-big search-bar"
+      data-qa="big"
+    >
+
+      <input
+        type="text"
+        class="form-search-big"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form
+      action="Post"
+      class="form-small bar"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        class="form-search-small search-bar"
+        data-qa="keypress"
```

Comment:
Add attribute data-qa="keypress" to input in the big search form not in small search form